packages/features/ee/workflows/lib/reminders/providers/emailProvider.ts

```diff
@@ -8,6 +8,7 @@ type EmailData = Omit<WorkflowEmailData, "to"> & {
 
 export async function sendOrScheduleWorkflowEmails(mailData: EmailData) {
   if (mailData.sendAt) {
+    if (mailData.sendAt <= new Date()) return;
```

Comment:
What is the timezone of `mailData.sendAt` ?  new Date() would return current date depending on the machine's timezone

Reply:
It doesn't matter, timezone info is forgotten when doing a numeric comparison (which this does)

12:00 UTC+1 === 11:00 UTC+0